niworkflows/interfaces/mni.py

```diff
@@ -231,18 +354,115 @@ def _validate_results(self):
 
 
 def mask(in_file, mask_file, new_name):
```

Comment:
Better more explicit about the action:

```Python
def apply_mask(....
```

Reply:
As a pre-existing funtion, I'm going to say this is out-of-scope for this PR. If we're going to bother renaming, we should probably just go ahead and use nilearn to handle masking.